<launch>

    <arg name="robot_run"                    default="false" />

    <!-- lingao depth camera driver bringup -->
    <include file="$(find lingao_visual)/launch/lingao_depthcamera.launch"/>
    
    <!-- lingao line_follower -->
    <node pkg="lingao_follower" name="lingao_line_follow_node" type="lingao_line_follow_node" output="screen">
          <param name="H_Max" value="68"/>
          <param name="H_Min" value="48"/>
          <param name="S_Max" value="255"/>
          <param name="S_Min" value="62"/>
          <param name="V_Max" value="255"/>
          <param name="V_Min" value="106"/>

          <param name="speed" value="0.3" />
          <param name="robot_run" value="$(arg robot_run)" />
          
    </node>

</launch>

<!-- RED          :  H:[318 - 360] S:[43 - 255] V:[46 - 255] -->
<!-- YELLOW :   H:[68   -  48]   S:[62 - 255] V:[106 - 255] -->
<!-- BLUE       :   H:[200 - 248] S:[68 - 255] V:[97 - 255] -->